noxfile.py

```diff
@@ -103,16 +103,15 @@ def lint(session):
 
 
 @nox.session(python=["3.8", "3.9", "3.10"])
-@nox.parametrize("pandas_version", ["1.5.0"])
-def test(session, pandas_version: str):
+def test(session):
     session.install(
         "-r",
         "requirements-dev.txt",
         "--timeout",
         "1500",
     )
     session.install(".")
-    session.run("python", "-m", "pip", "install", f"pandas~={pandas_version}")
+    session.run("python", "-m", "pip", "install", "-r", "requirements.txt")
```

Comment:
Do we need this?   we are already installing `requirements-dev.txt` in the previous line.